labgrid/resource/udev.py

```diff
@@ -32,7 +32,7 @@ def on_resource_added(self, resource):
         devices.match_subsystem(resource.match['SUBSYSTEM'])
         for device in devices:
             if resource.try_match(device):
-                break
+                self.log.debug(" matched successfully against %s", resource.device)
```

Comment:
Leading space?

Reply:
This is an indentation. The intention here is that the log message "belongs" to the log message in `try_match()`which is indented, too.